admin_app/src/app/integrations/page.tsx

```diff
@@ -74,35 +80,32 @@ const KeyManagement = ({
   return (
     <Layout.FlexBox
       key={"key-management"}
-      flexDirection={"column"}
-      sx={{
-        display: "flex",
-        alignSelf: "center",
-        px: sizes.baseGap,
-      }}
-      gap={sizes.baseGap}
+      flexDirection="column"
+      sx={{ maxWidth: 690, marginInline: 10 }}
+      gap={sizes.doubleBaseGap}
     >
-      <Typography variant="h5" align="center" color="primary">
-        API Key Management
+      <Typography variant="h5" color="primary" style={{ fontWeight: "bold" }}>
+        Your API Key
       </Typography>
       <Layout.FlexBox
-        flexDirection={"row"}
-        sx={{
-          display: "flex",
-          alignSelf: "center",
-          px: sizes.baseGap,
-        }}
-        gap={sizes.baseGap}
+        flexDirection="column"
+        // alignItems="center"
+        justifyContent="space-between"
+        gap={sizes.doubleBaseGap}
       >
+        <Typography variant="body1">
+          You will need your API key to interact with AAQ from your chat
+          manager. You can generate or re-generate your key here.
```

Comment:
Do we want to be specific and say. "You can generate a new key here. All old keys are invalidated when you create a new key"?

Reply:
Good idea, updating